Resources/views/Collector/statsd_prometheus.html.twig

```diff
@@ -19,9 +19,9 @@
 
 {% block menu %}
     <span class="label">
-    <span class="icon">{% spaceless %}
+    <span class="icon"><img ... /></span>
```

Comment:
that wasn't exactly what I meant 😓

Reply:
:) i click on the commit suggested change before really reading your changes, i fixed the code now